docs/classes/universal-profile.md

```diff
@@ -320,9 +322,10 @@ Available as a static or non-static method callable on the LSPFactory library in
 
 #### 1. `profileData` - Object
 
-Object containing the [LSP3 Metadata](../../../standards/universal-profile/lsp3-profile-metadata) fields which will be processed and uploaded to IPFS.
+Object containing the [LSP3 Metadata](../../../standards/universal-profile/lsp3-universal-profile-metadata.md) fields which will be processed and uploaded to IPFS.
```

Comment:
These ones were updated by @YamenMerhi here https://github.com/lukso-network/tools-lsp-factory/pull/198/files
Can we remove these reverts here? These should be correct

Reply:
I don't see the initial file in the docs repo, I only see this one:

https://github.com/lukso-network/docs/blob/main/docs/standards/universal-profile/lsp3-universal-profile-metadata.md

Reply:
<img width="462" alt="image" src="https://github.com/lukso-network/tools-lsp-factory/assets/477945/123b8bd9-9987-448c-b612-fe9ffb3661b2">

Reply:
Nice catch, let's merge the other PR first then:

https://github.com/lukso-network/docs/pull/593/files

Reply:
Fixed